docs/public/samples/z-up.html

```diff
@@ -56,18 +56,23 @@
       document.body.appendChild(renderer.domElement);
 
       const [threeModel, threeModelAnimation] = loadThreeModel(scene, renderer);
-      // Adjust rotation for the z-up
+      // Adjust rotations for the Z-up coordinate system
       threeModel.rotation.x = Math.PI / 2;
-      threeModel.rotation.y = Math.PI;
       scene.add(threeModel);
 
+      // Init OrbitControls first
+      const controls = new OrbitControls(camera, renderer.domElement);
+      
       // Init Gizmo with OrbitControls
       const gizmo = new ViewportGizmo(camera, renderer, getGizmoConfig());
-      gizmo.attachControls(new OrbitControls(camera, renderer.domElement));
+      gizmo.attachControls(controls);
 
       // Optional. Set the target
       gizmo.target.set(0, 0, 3);
       camera.lookAt(gizmo.target);
+      
+      // Controls must be updated after the camera transform - https://threejs.org/docs/#examples/en/controls/OrbitControls.update
+      controls.update();
```

Comment:
This was the critically missing piece for "initialization" failures discussed in other issues.